homeassistant/components/saj/sensor.py

```diff
@@ -162,14 +167,18 @@ def remove_listener():
 class SAJsensor(Entity):
     """Representation of a SAJ sensor."""
 
-    def __init__(self, pysaj_sensor):
+    def __init__(self, pysaj_sensor, inverter_name=""):
```

Comment:
Make `None` the default instead of an empty string.

Reply:
Done